types/akumina-core/index.d.ts

```diff
@@ -0,0 +1,1471 @@
+// Type definitions for Akumina 4.5
+// Project: https://github.com/akumina/AkuminaDev
+// Definitions by: Akumina <https://github.com/akumina>
+//                 Jason Arden <https://github.com/jasonarden>
+//                 Sean Glover <https://github.com/MarshHawk>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+/// <reference types="jquery" />
+
+// tslint:disable-next-line interface-name
+import IGetListRequest from "./interfaces/IGetListRequest";
+// tslint:disable-next-line interface-name
+import ISavePageWidgetRequest from "./interfaces/ISavePageWidgetRequest";
+
+declare namespace Akumina {
+    class BaseWidget {
+        constructor();
+        Properties: any;
+        GetPropertyValue(requestIn: any, key: string, defaultValue: any): any;
+        RefreshWidget(newProps: any): void;
+        BindTemplate(templateUri: string, data: any, targetDiv: string): void;
+    }
+
+    class Expression {
+        And(expression: any): Expression;
+        Or(expression: any): Expression;
+        EqualTo(expression: any): Expression;
+        In(expression: any): Expression;
+        NotEqualTo(expression: any): Expression;
+        GreaterThan(expression: any): Expression;
+        GreaterThanOrEqualTo(expression: any): Expression;
+        LessThan(expression: any): Expression;
+        LessThanOrEqualTo(expression: any): Expression;
+        Contains(expression: any): Expression;
+    }
+
+    class PropertyExpression extends Expression {
+        constructor(param?: any);
+    }
+    // class OrExpression extends Expression { }
+    // class EqualExpression extends Expression { }
+    // class InExpression extends Expression { }
+    // class NotEqualExpression extends Expression { }
+    // class GreaterThanExpression extends Expression { }
+    // class GreaterThanOrEqualExpression extends Expression { }
+    // class LessThanExpression extends Expression { }
+    // class LessThanOrEqualExpression extends Expression { }
+    // class ContainsExpression extends Expression { }
+    class SortDirection {
+        static Descending: string;
+    }
+
+    namespace AppParts {
+        class Banner {
+            constructor(options: any);
+        }
+        class Traffic {
+            constructor(options: any);
+        }
+        class Calendar {
+            constructor(options: any);
+        }
+        class CompanyCalendar {
+            constructor(options: any);
+        }
+    }
+
+    namespace Digispace {
+        class ConfigurationContext {
```

Comment:
I assume you know from the lint messages that a class with only static members doesn't make much sense but prefer to keep it this way.

Reply:
yes, yes, just deferring that improvement for the moment, thanks.